streams/src/test/java/org/apache/kafka/streams/perf/SimpleBenchmark.java

```diff
@@ -153,14 +199,16 @@ private Properties setJoinProperties(final String applicationId) {
      * KStream record joins to exactly one element in the KTable
      */
     public void kStreamKTableJoin(String kStreamTopic, String kTableTopic) throws Exception {
-        CountDownLatch latch = new CountDownLatch(numRecords);
+        CountDownLatch latch = new CountDownLatch(1);
```

Comment:
Oh, just an optimization. Instead of counting down the latch for every record, we just count it down when we see the last key.